api_tests/reviews/mixins/filter_mixins.py

```diff
@@ -55,7 +56,7 @@ def providers(self):
     def all_actions(self, providers):
         actions = []
         for p in providers:
```

Comment:
I'm not a huge fan of one letter variables but ultimately it's up to you.

Reply:
I agree here.